Tests/LoggingTests/Test+Floats.swift

```diff
@@ -19,7 +19,11 @@ final class FloatTests: XCTestCase {
         logger.debug("\(max, privacy: .public)")
         XCTAssertEqual(logging.recorder.message, "\(CGFloat.greatestFiniteMagnitude)")
 
+        #if swift(<5.5)
+        logger.debug("\(Float(value), format: .fixed(precision: 2), privacy: .public)")
```

Comment:
Hmm this satisfies the compiler but not the test. We shouldn't be correcting the test but rather the code this is testing. I'll review the actual problem and see what's going on. I'll leave this PR open for now.

Reply:
Yep. Smallest change path.

I did it the other way but that involved more code and then ran into the precision issue.
I'll submit a separate PR with that version since it is the more complete fix.

Reply:
Yep. Smallest change path.

I did it the other way but that involved more code and then ran into the precision issue.
I'll submit a separate PR with that version since it is the more complete fix.